docs/xdr/FAQ/Alerts_qa.md

```diff
@@ -1,7 +1,22 @@
-## Alert date
+## Alert date and additional events
 
 When an alert is triggered, additional events can enrich this alert but the date of the alert will not be updated (date= 1st trigger). 
 
+## Alert and events timestamp relation
+
+There are 4 informations available on SEKOIA.IO:
+1- `log timestamp`:            Generated by the machine and present in the message (no actions done by SEKOIA.IO)
+2- `collector timestamp`:      Period when the log has been processed by the collector (timestamp collected from the endpoint available in the log unless it is missing so the timestamp of the header generated by the collector will be used instead)
+3- `event.created`:            Period when the log has been arrived in SEKOIA.IO
+4- `Alert date created`:       Period when the alert has been generated in SEKOIA.IO
+
+The chronological time should be as follow (from old to recent):
+`log timestamp (1) < collector timestamp (2) < event.created (3) < Alert created (4)`
+
+!!! Troubleshoot tips
+    Please check the different dates and collector configuration
```

Comment:
is there a link to the collector configuration?

Reply:
It is on client side when configuring their collector

Reply:
@ka0ula From Charles comments, I have added as optional as it depends it only applies when a collector is set.